src/Converter.cc

```diff
@@ -172,6 +172,17 @@ void Converter::ConvertDescendantsImpl(tinyxml2::XMLElement *_e,
   }
 }
 
+
+/////////////////////////////////////////////////
+// TODO(jenn) delete, used for debugging
+std::string PrintElement(const tinyxml2::XMLElement *_elem)
```

Comment:
nit Perhaps this is useful to keep around? Perhaps implement using a trace or debug logging level?

Reply:
`PrintElement` is now `ElementToString` put in `XmlUtils.cc`: 9ed27130e8343867e79d88dae4be3894a7f0ac82